### PR TITLE
[545] bug header spacing bug

### PR DESCRIPTION
**Pull request by stellenberger**
### Context

- [This trello ticket](https://trello.com/c/KMrL2eOn/545-bug-header-spacing-bug)

### Changes proposed in this pull request

- Comment out the navigation bar for time being (until more items are introduced)
- Fix the spacing issue on the header

### Checklist

- [x] Make sure all information from the Trello card is in here
- [x] Attach to Trello card
- [x] Rebased master
- [x] Cleaned commit history
- [x] Tested by running locally

### CODE REVIEW

File: app/components/header/style.scss [hunk 1]
```diff
@@ -38,3 +38,7 @@
     border-bottom-color: govuk-colour("purple");
   }
 }
+
+#navigation {
```
Comment (darokel) [inline app/components/header/style.scss:42]: I think we can just use a govuk utility class to zero out the margin `govuk-!-margin-bottom-0` on the element or something. Also we should probably avoid targeting id selectors due to the specificity

**Review by darokel — CHANGES_REQUESTED** [review]

**Comment by stellenberger** [comment]
[[Bug] Header spacing bug](https://trello.com/c/KMrL2eOn/545-bug-header-spacing-bug)

**Comment by stellenberger** [comment]
[Remove the gray NavigationBar with one item from Support Console](https://trello.com/c/BZWIN6gl/549-remove-the-gray-navigationbar-with-one-item-from-support-console)

**Review by darokel — APPROVED** [review]
Not sure why that database diagram is being added. Could we remove it from the diff?

**Pull request merged**